tests/nbody-gradient/input.dat

```diff
@@ -0,0 +1,28 @@
+#! MP2/aug-cc-pv[DT]Z many body gradient of an arbitrary Helium complex
+
+comparison_gradient_dict = {} #TEST
+comparison_gradient_dict['nocp'] = [[-0.000000000000,     0.000000000000,    -0.000008106671], #TEST  
+                                    [ 0.000000000000,    -0.000000478704,     0.000004053336], #TEST
+                                    [-0.000000000000,     0.000000478704,     0.000004053336]] #TEST
+
+comparison_gradient_dict['cp']   = [[-0.000000000000,     0.000004675134,     0.000001145535], #TEST
+                                    [ 0.000002267167,    -0.000002253992,    -0.000000572767], #TEST
+                                    [-0.000002267167,    -0.000002421141,    -0.000000572767]] #TEST
+
+comparison_gradient_dict['vmfc'] = [[ 0.000000000000,    -0.000000000000,    -0.000005200318], #TEST
+                                    [ 0.000000000000,    -0.000000478704,     0.000002578173], #TEST
+                                    [-0.000000000000,     0.000000478704,     0.000002622146]] #TEST
+
```

Comment:
These numbers seem to jump around quite a bit, can we try a larger basis to see if they converge?

Also the `cp` results are bit odd, Im not quite sure we should be getting a gradient in the `x` direction. Best to double check.

Reply:
It seems the ```cp``` results have an interesting error. Depending on the recognized point group symmetry for the individual nbody computations, Psi4 will transform the coordinates differently and inconsistently. Adding the gradients then will give wrong results. See for example the following two geometries from the test output file:
```
  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass
    ------------   -----------------  -----------------  -----------------  -----------------
          HE(Gh)      0.000000000000     1.885618083164     0.000000000000     4.002603254150
          HE(Gh)      2.828427124746    -0.942809041582     0.000000000000     4.002603254150
          HE         -2.828427124746    -0.942809041582     0.000000000000     4.002603254150
```

``` 
==> Geometry <==

    Molecular point group: c2v
    Full point group: C2v

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass
    ------------   -----------------  -----------------  -----------------  -----------------
          HE          0.000000000000     0.000000000000    -1.885618083164     4.002603254150
          HE(Gh)      0.000000000000    -2.828427124746     0.942809041582     4.002603254150
          HE(Gh)     -0.000000000000     2.828427124746     0.942809041582     4.002603254150
```

@dgasmith, do you suggest that we force the symmetry to be c1 in the nbody_driver and/or use ```no_reorient``` and ```no_com```?

Reply:
I think this is one of the hazards of taking n_body from energy-->gradient -- orientations start to matter. Unless @dgasmith thinks n_body can be a lot smarter than I'm supposing, I agree with `no_com`/`no_reorient`.

Reply:
Sounds good. I'll use ```no_reorient```/```no_com``` for the moment and see if I can do something for the rotation later. @loriab, does not a development version of QCDB have alignment capability that takes into account symmetry/stereochemistry?

Reply:
Yes, even the current-in-p4 qcdb has very possibly good enough alignment capability for this purpose. future pass, though. And it'd need a balance to make sure the alignment wasn't slowing down the n_body.